File: pay/src/main/java/com/truenine/component/pay/api/model/request/QueryOrderApiRequestParam.java
package com.truenine.component.pay.api.model.request;

import jakarta.validation.constraints.Size;
import lombok.Data;

@Data
public class QueryOrderApiRequestParam {

  @Size(min = 6, max = 32, message = "参数长度不对")
  private String orderId;

  @Size(min = 1, max = 32, message = "参数长度不对")
  private String orderNo;

}

File: pay/src/main/kotlin/net/yan100/compose/pay/models/request/QueryOrderApiRequestParam.kt
package net.yan100.compose.pay.models.request

import jakarta.validation.constraints.Size


open class QueryOrderApiRequestParam {
  open var orderId: @Size(min = 6, max = 32, message = "参数长度不对") String? = null
  open var orderNo: @Size(min = 1, max = 32, message = "参数长度不对") String? = null
}
